<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="header">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="berandaButton">
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Beranda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="userButton">
   <property name="geometry">
    <rect>
     <x>1600</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="matkulButton">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Matkul</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../logo_siak.qrc"/>
 </resources>
 <connections/>
</ui>
